README.md

```diff
@@ -45,7 +45,9 @@ guide the learner’s interaction with the component.
 **mobileInstruction** (string): This is optional text that will be substituted for **instruction** when `Adapt.device.screenSize` is `small` (i.e., when viewed on mobile devices).  
 
 **_setCompletionOn** (string): This value determines when the component registers as complete. Acceptable values are `"allItems"` and `"inview"`. `"allItems"` requires each pop-up item to be visited. `"inview"` requires the **Hot Graphic** component to enter the view port completely.  
-  
+
+**_useGraphicsAsPins** (boolean): When set to 'true', graphics image specified under attribute **_graphic** of **_items** attribute will appear as pins. The default is `false`.
```

Comment:
```suggestion
**_useGraphicsAsPins** (boolean): If set to `true`, the image specified by **_graphic.src** will be ignored and the popup images specified in **_items[n]._graphic.src** will instead be laid out in a grid system as defined by the classes in hotgraphic.less. See [example.json](example.json#L74-L161) for a working example..
```

Reply:
@moloko we need "The default is `false`" in the end right ?

Reply:
yes might as well for consistency & clarity

Reply:
@sarveshwar-gavhane, yes.

Reply:
I think if updating the documentation it should probably mention that this works in conjunction with the `_items[n]._classes` property, e.g. `rows-1 cols-2`, etc.

Reply:
@brian-learningpool my updated version does reference both the custom classes and links to example.json which demos the use of them

Reply:
Hi @moloko, that's on the `_useGraphicsAsPins` flag though.  It might we worthwhile to add some corresponding help text to the `_items._classes` property?

Reply:
Yes, good idea. Especially seeing as that property isn't even mentioned currently! Maybe something along the lines of:
"Used to specify custom CSS classes to be applied to the popup item. If **_useGraphicsAsPins** is `true`, used to specify the layout using the 'rows' and 'cols' classes defined in hotgraphic.less"
?

Reply:
are we adding help text for _items._classes in this PR or in separate ticket.

Reply:
in this PR is fine, thanks

Reply:
Yes please.

Reply:
`_items[n]._graphic._classes` isn't mentioned currently, kindly suggest appropriate help text for the same.
Link : https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/master/example.json#L33